app/src/main/res/layout/previous_insurer_item_layout.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Could this be a `TextView` with a compound drawable? 🙂

Reply:
Maybe! Can I pass a `TextView` to glide in order to load the logo?

Reply:
It's possible, albeit not as easy, I suppose: https://stackoverflow.com/a/55105155

That might be enough reason to not go for a compound drawable in this case. I trust your judgment on whichever way you decide to go on this issue 🙂